docs/content/docs/dev/table/types.md

```diff
@@ -1487,6 +1481,88 @@ Not supported.
 {{< /tab >}}
 {{< /tabs >}}
 
+Casting
+-------
+
+Flink SQL can perform casting between a defined pair `from` type and `target` type pair. While some
```

Comment:
```suggestion
Flink SQL can perform casting between a defined `from` type pair and `target` type pair. While some
```

Reply:
> defined `from` type pair and `target` type pair

This doesn't make sense, as it's one pair `(from, target)`. This wording sounds like there are two pairs, one called `from` and the other called `target`

Reply:
That's then the confusing part for me because it has `pair` twice in the original sentence. So we could completely omit `pair` and say something like ```Flink Table API and SQL can perform casting between a defined `from` type and `target` type.```  ? Or ```Flink Table API and SQL can perform casting between a defined `from` type and `target` type pair.```

Reply:
Ok will remove the pair once for all then :) check now